docs/developer-guide/ksqldb-reference/scalar-functions.md

```diff
@@ -1199,6 +1199,33 @@ TIMEZONE is an optional parameter and it is a `java.util.TimeZone` ID format, fo
 "America/Los_Angeles", "PDT", "Europe/London". For more information on timestamp formats, see
 [DateTimeFormatter](https://cnfl.io/java-dtf).
 
+### `FORMAT_TIME`
+
+Since: 0.20
+
+```sql
+FORMAT_TIME(timestamp, 'HH:mm:ss.SSS')
```

Comment:
isn't this `format_time(time, ...)`? Same for the`timestamp` word in the below paragraph.